packages/openapi-typescript/examples/digital-ocean-api.ts

```diff
@@ -3984,7 +3984,7 @@ export interface external {
        */
       type?: "custom" | "lets_encrypt";
     };
-    /** Custom Certificate Request */    certificate_request_custom: certificate_create_base[] & {
+    /** Custom Certificate Request */    certificate_request_custom: external["resources/certificates/models/certificate_create.yml"]["certificate_create_base"] & {
```

Comment:
This was generating incorrectly before due to an unrelated bug. Fix was easy; we just didn’t have a test for it before.